cli/cmd/config.go

```diff
@@ -22,66 +24,124 @@ var configCmd = &cobra.Command{
 var setConfigCmd = &cobra.Command{
 	Use:   "set <property> <value>",
 	Short: "Set a configuration property in Odigos",
-	Args:  cobra.ExactArgs(2),
+	Args:  cobra.MinimumNArgs(2),
 	Run: func(cmd *cobra.Command, args []string) {
 		property := args[0]
-		value := args[1]
+		value := args[1:]
 
 		ctx := cmd.Context()
 		client := cmdcontext.KubeClientFromContextOrExit(ctx)
 		ns, _ := cmd.Flags().GetString("namespace")
 		if ns == "" {
-			ns = "odigos-system"
+			ns = consts.DefaultOdigosNamespace
 		}
```

Comment:
please use

```
ns, err := resources.GetOdigosNamespace(client, ctx)
```

To automatically get the namespace instead of having the user supplying it